### PR TITLE
Move GUI dependent code from VideoUtils to VideoGUIUtils

### PR DESCRIPTION
**Pull request by notspiff**
## Description
Split out GUI dependent code in VideoUtils to separate file

## Motivation and context
Better separation

## How has this been tested?
It builds

## What is the effect on users?
None

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [x] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **Student submission** (PR was done for educational purposes and will be treated as such)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Review by enen92 — APPROVED** [review]
Thanks much

**Review by ksooo — COMMENTED** [review]
Shouldn't video/VideoGUIUtils.h rather be located in video/guilib ? I thought we "invented" video/guilb exactly to have a dedicated home for GUI dependent video code.

File: xbmc/interfaces/builtins/PlayerBuiltins.cpp [hunk 1]
```diff
@@ -425,7 +426,7 @@ namespace
 void GetItemsForPlayList(const std::shared_ptr<CFileItem>& item, CFileItemList& queuedItems)
 {
   if (VIDEO_UTILS::IsItemPlayable(*item))
-    VIDEO_UTILS::GetItemsForPlayList(item, queuedItems);
+    GetItemsForPlayList(item, queuedItems);
```
Comment (ksooo) [inline xbmc/interfaces/builtins/PlayerBuiltins.cpp:429]: Doesn't this now call itself recursively? I think the namespace needs to be kept to call the right function.

File: xbmc/utils/PlayerUtils.cpp [hunk 2]
```diff
@@ -31,7 +31,7 @@ bool CPlayerUtils::IsItemPlayable(const CFileItem& itemIn)
     return true;
 
   // Movies / TV Shows / Music Videos
-  if (VIDEO_UTILS::IsItemPlayable(item))
+  if (IsItemPlayable(item))
```
Comment (ksooo) [inline xbmc/utils/PlayerUtils.cpp:34]: Doesn't this now call itself recursively? I think the namespace needs to be kept to call the right function.

**Review by ksooo — CHANGES_REQUESTED** [review]
I think this would have not passed a runtime test successfully. ;-)

File: xbmc/interfaces/builtins/PlayerBuiltins.cpp (hunk 1 above)
Comment (notspiff) [reply xbmc/interfaces/builtins/PlayerBuiltins.cpp]: yeh. bit too blind faith in clangd suggestions.

File: xbmc/utils/PlayerUtils.cpp (hunk 2 above)
Comment (notspiff) [reply xbmc/utils/PlayerUtils.cpp]: indeed.

**Review by ksooo — APPROVED** [review]

**Comment by ksooo** [comment]
Two approvals, build is green. Going to merge...

**Pull request merged**